stage3/react/modules/module04/README.md

```diff
@@ -0,0 +1,42 @@
+# Неделя 4
+
+## React. Hooks
+
+
+
+
+!! Задание выполняется в личном приватном репозитории !!
+Нужно выполнять задание на базе предыдущего
+
+
+
+
+Что должно быть сделано:
+
+
+
+1) Создать отдельную ветку для этого задания
+2) Переписать основную страницу на хуки сохранив функционал. Все тесты по итогу должны работать
+3) Переписать форму на React Hook Form, сохранив функционал. Все тесты по итогу должны работать
+
+
+### Оценка
+
+
+
+
+Задание будет оцениваться ментором в марте. Для удобства проверки создайте Pull Request(**МЕРЖИТЬ НЕ НАДО**).
+
+
+
+
+1) Hooks - **15 баллов**
+
+
+
+
+Если хоть один из пунктов не выполнен, то задание оценивается в **0 баллов**.
+
+### Теория
+https://reactjs.org/docs/hooks-intro.html
+https://react-hook-form.com/api/
```

Comment:
Introducing Hooks: https://reactjs.org/docs/hooks-intro.html
React hooks: https://www.youtube.com/watch?v=3-Zh_DAzCi0
React Course - Beginner's Tutorial for React JavaScript Library: https://www.youtube.com/watch?v=bMknfKXIFA8
react-hook-form: https://react-hook-form.com/api/
react-hook-form tutorial: https://www.youtube.com/watch?v=bU_eq8qyjic

Reply:
React-Hooks: https://www.youtube.com/watch?v=FAhnawACrOg&list=PLzLiprpVuH8cuG8ijG_m0-y63B3suk4vu&index=4&ab_channel=RollingScopesSchool